### ISSUE TITLE
Different Algorithms for SecretKey and Cipher

### ISSUE THREAD

**Issue by agrebnev** [simbiose/Encryption#7]
Hi @ademar111190 

Please add possibility to specify different algorithms for SecretKey and Cipher.

**Comment by ademar111190** [comment]
Yes it is in the plans, today change the algorithms is almost impossible but in next versions the goal is a good modularization to change easily the algorithms.

**Comment by agrebnev** [comment]
I suppose it is easy. Now you have setAlgorithm method. I offer to make one more like setCipherAlgorithm.

I would like to use "AES/CBC/PKCS5Padding" for Cipher and "AES" for Secret Key. Now it is impossible.
